File: Java/Day 3/Question2Calculation.java
//Write an application that input three integers from the user and displays the sum average product smallest largest of he number 
//colllect three numbers sum smallest largest average and product

import java.util.Scanner;
public class SquareCalculation{
    public static void main(String[]args){
    Scanner input = new Scanner(System.in);
  
   
    System.out.print("Enter the first number: ");
    int number1 = input.nextInt();

    System.out.print("Enter the second number: ");
    int number2 = input.nextInt();
    
    System.out.print("Enter the third number: ");
    int number3 = input.nextInt();

    int largest = number1;
    int smallest = number1;
    

    int sum = number1 + number2 + number3;

    int average = sum / 3;

    int product = number1 * number2 * number3;


    if (number2 > number1 && number3 > number1){
    largest = number2;
            
    }
    else if (number3 > number1 && number3 > number1){
    largest = number3;
            
    }
    
    if (number2 < number1 && number3 < number1){
    smallest = number2;
            
    }
    else if (number3 < number1 && number3 < number1){
    smallest = number3;
            
    }

    System.out.println("The smallest number is " + smallest);

    System.out.println("The largest number is " + largest);
    
    System.out.println("The sum of the numbers is " + sum);

    System.out.println("The average number is " + average);

    System.out.println("The product number is " + product);


    }
}

File: JavaScript/Day 3/Question2Calculation.js
const prompt = require("prompt-sync")({sigint: true});
   
    let number1 = parseInt(prompt("Enter the first number: "));
 
    let number2 = parseInt(prompt("Enter the second number: "));
    
    let number3 = parseInt(prompt("Enter the third number: "));
    
    

    let largest = number1;
    let smallest = number1;
    
    let sum = number1 + number2 + number3;

    let average = sum / 3;

    let product = (number1 * number2) * number3;

    if (number2 > number1 && number3 > number1){
    largest = number2;
            
    }
    else if (number3 > number1 && number3 > number2){
    largest = number3;
            
    }
    
    if (number2 < number1 && number3 < number1){
    smallest = number2;
            
    }
    else if (number3 < number1 && number3 < number1){
    smallest = number3;
            
    }

    console.log("The smallest number is " + smallest);

    console.log("The largest number is " + largest);
    
    console.log("The sum of the numbers is " + sum);

    console.log("The average number is " + average);

    console.log("The product number is " + product);


